kuma/static/js/newsletter.js

```diff
@@ -5,23 +5,18 @@
 
     var newsletterForm = doc.getElementById('newsletterForm');
     var isHidden = false;
-    var isArticle = false;
+    var isArticle = window.location.pathname.indexOf('/docs/') !== -1;
```

Comment:
The changes in this file are mostly a straightforward port of jquery functions to modern standards. Note that these changes affect both the wiki and the beta sites, so any errors here will affect the live site.

This particular line changes the way we detect whether we're on an article page to be compatible with React-based article pages.